crypto/crypto_aesctr_aesni.c

```diff
@@ -60,7 +64,11 @@ crypto_aesctr_aesni_stream_wholeblocks(struct crypto_aesctr * stream,
 		be64enc(block_counter_be_arr, block_counter);
 
 		/* Encrypt the cipherblock. */
+#ifdef NO_LOADU_SI64
+		bufsse = _mm_loadu_si128((const __m128i *)block_counter_be_arr);
```

Comment:
This performs a 128-bit load from an 8-byte array, which seems a bit optimistic.

Reply:
ah right, I was only thinking about stream->pblk.